motioneye/utils/__init__.py

```diff
@@ -474,9 +473,9 @@ def build_editable_mask_file(
     im = Image.new('L', (width, height), 255)  # all white
     dr = ImageDraw.Draw(im)
 
-    for y in numpy.arange(ny):
```

Comment:
`numpy.arange()` is a parallel / vector operation while Python's built-in `range()` is a serial operation but I doubt that matters unless performance is vital.

https://realpython.com/how-to-use-numpy-arange/#comparison-of-range-and-nparange

Reply:
Thanks for the hint. I think it is not critical here, this is only done once when editing the mask and once when saving the mask, while on actual motion detection generated `/etc/motioneye/mask_X_privacy.pgm` is used as overlay. But I'll add some debug code locally to verify this.